app/src/question/index.js

```diff
@@ -79,11 +87,14 @@ module.exports = rust.class({
     var ctx = this;
     return rust.o2([
       'question',
-      {className: isOpen ? 'open' : 'closed'},
+      {className: [
+        isOpen ? 'open' : 'closed',
+        (q.councilToken === councilToken) ? 'myAsk' : 'didntAsk'
```

Comment:
For consistency's sake, I use hyphens in my classes. So 'my-ask' and 'didnt-ask'.
For even more consistency I like to just use one class when its "additive". Like if theres the regular style then one exception style. Which might looks like `q.token === token ? 'mine' : ''`.